src/Nox.Lib/Application/Commands/CommandBase.cs

```diff
@@ -38,4 +38,12 @@ protected Entity GetEntityDefinition<E>()
     /// <param name="request"></param>
     /// <param name="cancellationToken"></param>
     protected virtual Task OnCompletedAsync(TRequest request, TEntity entity) { return Task.CompletedTask; }
+    
+    /// <summary>
+    /// Command handler completed
+    /// Use this method to override, update, validate or other run custom logic regarding the batch process of the updated/created/deleted Entities
+    /// </summary>
+    /// <param name="request"></param>
+    /// <param name="entities"></param>
+    protected virtual Task OnBatchCompletedAsync(TRequest request, IEnumerable<TEntity> entities) { return Task.CompletedTask; }
```

Comment:
@msadikkose check this one please and align your code with it
https://github.com/NoxOrg/Nox.Generator/pull/1201/files#diff-cfc7e0642eb260adb2c4439fa9b745dbd702d2870a7df947e3d67df369e818ba